hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCleanConfig.java

```diff
@@ -67,6 +63,31 @@ public class HoodieCleanConfig extends HoodieConfig {
       .withDocumentation("Only applies when " + AUTO_CLEAN.key() + " is turned on. "
           + "When turned on runs cleaner async with writing, which can speed up overall write performance.");
 
+  public static final ConfigProperty<String> CLEANER_COMMITS_RETAINED = ConfigProperty
```

Comment:
Ordering needed to change so the key and alternatives could be referenced in the inference below